whisper_functions.py

```diff
@@ -0,0 +1,50 @@
+import whisper
+import os
+import torch
+import pandas as pd
+
+def transcribe(model, input_file: str, cuda: bool = False):
+    """
+    Transcribe the given audio using the given Whisper model. \n
+    :param model: The Whisper model to use for transcription. \n
+        Can be loaded using whisper.load_model("turbo").to(device) \n
+    :param input_file: The path to the audio file to transcribe. \n
+    :param cuda: Whether to use CUDA for transcription. \n
+    """
+
+    print("Transcribing: ", input_file)
+    try:
+        result = model.transcribe(
+            input_file,
+            fp16=True if cuda else False,
+            language="en"
+        )
+    except Exception as e:
+        print("Error: ", e)
+        return "Error"
+
+    return result["text"]
+
+if __name__ == "__main__":
+    device = 'cuda' if torch.cuda.is_available() else 'cpu'
+    # Load the model
+    model = whisper.load_model("turbo").to(device)
+
+    audio_dir = "data/archive/Raw JL corpus (unchecked and unannotated)/JL(wav+txt)"
```

Comment:
A bit hard coded, but not sure what more we can do now. Think it's a good idea to break this out now into a function or shall we leave it till we are more done with the RL model?

Reply:
Allt är hårdkodat för att pröva det som ett exempel. Filen är framförallt för funktioner kopplade till Whisper

Reply:
I like the new changes.